src/server/personlookup.js

```diff
@@ -0,0 +1,83 @@
+'use strict';
+
+const request = require('request');
+const authSupport = require('./authsupport');
+
+let navConfig = null;
+let cachedAccessToken = null;
+
+const init = config => {
+    navConfig = config;
+    cachedAccessToken = null;
+};
+
+const hentPerson = async aktørId => {
+    return new Promise((resolve, reject) => {
+        logonToNav()
+            .then(jwt => {
+                request.get(
```

Comment:
Skal det ikke være `await request.get`?

Reply:
> Skal det ikke være `await request.get`?

Hmm, request returnerer jo ikke et promise 😛 De har laget en alternativ pakke, request-promise-native, som vi kunne ha brukt og spart oss for litt håndkoding av promises, men det er mulig det ikke er verdt det :)